### PR TITLE
feat(curriculum): create objects quiz

### LINKED ISSUE #56202: [HACKTOBERFEST 2024] - Objects Quiz
> [!IMPORTANT]
> This issue is part of our 2024 Hacktoberfest initiative. If you are interested in working on this issue, refer to our [top-level tasks](https://github.com/freeCodeCamp/freeCodeCamp/issues/56170) to see if this issue is claimed, or to claim it if it is not.

This issue is to create the Objects quiz.

The file you should be working on is found in `/curriculum/challenges/english/25-front-end-development/quiz-objects`.

**All quiz questions need to relate to one of these topics:**

- everything:tm: is an object in JS
- property access (dot notation vs bracket notation)
- nested property access
- Object() constructor
- optional chaining (?.) operator 
- destructuring (shorthand `const { prop } = obj;`)
- getters and setters
- prototype inheritance

> [!CAUTION]
> Questions which do not fall under these topics will be rejected.

You are welcome to include code blocks in your questions, but please keep them short (four lines or less).

Please do NOT open a pull request until you have completed all questions. Your quiz should have **exactly 20 questions**.

### PR DESCRIPTION
**Pull request by IAMOTZ**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes [#56202](https://github.com/freeCodeCamp/freeCodeCamp/issues/56202)

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-objects/66edcd0ecb4b25dc64a34804.md [hunk 1]
```diff
@@ -17,439 +17,464 @@ Answer all of the questions below correctly to pass the quiz.
...
-Placeholder question
+Which of the following correctly access the value of `key` property in the object below:
+```
+const obj = { key: "some-value" }
+```
 
 #### --distractors--
 
-Placeholder distractor 1
+`object -> name`
```
Comment (gikf) [inline curriculum/challenges/english/25-front-end-development/quiz-javascript-objects/66edcd0ecb4b25dc64a34804.md:71]:
```suggestion
`obj -> key`
```

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-objects/66edcd0ecb4b25dc64a34804.md [hunk 2]
```diff
@@ -17,439 +17,464 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 2
+`object#name`
```
Comment (gikf) [inline curriculum/challenges/english/25-front-end-development/quiz-javascript-objects/66edcd0ecb4b25dc64a34804.md:75]:
```suggestion
`obj#key`
```

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-objects/66edcd0ecb4b25dc64a34804.md [hunk 3]
```diff
@@ -17,439 +17,464 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 3
+`object[name]`
```
Comment (gikf) [inline curriculum/challenges/english/25-front-end-development/quiz-javascript-objects/66edcd0ecb4b25dc64a34804.md:79]:
```suggestion
`obj[key]`
```

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-objects/66edcd0ecb4b25dc64a34804.md [hunk 4]
```diff
@@ -17,439 +17,464 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 #### --answer--
 
-Placeholder answer
+`object.name`
```
Comment (gikf) [inline curriculum/challenges/english/25-front-end-development/quiz-javascript-objects/66edcd0ecb4b25dc64a34804.md:83]:
```suggestion
`obj.key`
```

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-objects/66edcd0ecb4b25dc64a34804.md [hunk 5]
```diff
@@ -17,439 +17,464 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 3
+`object?.key`
 
 #### --answer--
 
-Placeholder answer
+`object.?key`
```
Comment (gikf) [inline curriculum/challenges/english/25-front-end-development/quiz-javascript-objects/66edcd0ecb4b25dc64a34804.md:369]:
```suggestion
`object?.key`
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Review by gikf — APPROVED** [review]

**Comment by gikf** [comment]
Hi @IAMOTZ. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**